src/averageAges.js

```diff
@@ -36,8 +52,31 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+let mother = '';
+
+function callback(arr) {
```

Comment:
Consider renaming the function 'callback' to a more descriptive name like 'isMotherWithName'.